### ISSUE TITLE
iir: tests fail

### ISSUE THREAD

**Issue by dennisguse** [openitu/STL#55]
Failing `pcmdemo` tests:
* pcmdemo1-verify
* pcmdemo2-verify
* pcmdemo3-verify
* pcmdemo4-verify
* pcmdemo5-verify
* pcmdemo6-verify
* pcmdemo7-verify
* pcmdemo8-verify
* pcmdemo9-verify

Used Ubuntu 16.10 (64bit).
Patch status today.

[LastTest.log.txt](https://github.com/dennisguse/stl2009/files/992274/LastTest.log.txt)

**Comment by ludomal** [comment]
The original STL code was using ```sub``` to compare the signals with ```-equiv 1``` parameter. This allows to account for rounding of floating point values and validates the test vectors if they are within +/- 1 of the target value.
I have updated ```src/iir/CMakeLists.txt``` to make use of ```sub```. I have also improved ```sub``` so it returns an error code when the signals being compared are not equivalent. Before it would just exit with a code 0.
